components/FixedBanner.vue

```diff
@@ -93,6 +93,16 @@ export default {
             if (showHeader && this.header) {
               bannerContent = bannerHeader || {};
             } else if (showConsent && this.consent) {
+              if (bannerConsent.text && bannerConsent.text.length) {
```

Comment:
Can we abstract this logic to a separated function where we just get the consent content? It may help to make it more readable in the future.